connector.go

```diff
@@ -202,6 +192,24 @@ func (wrb *WapiRequestBuilder) BuildBody(obj IBObject) []byte {
 		return nil
 	}
 
+	if t == GET && len(eaSearch) > 0 {
+		payload := make(map[string]string)
+		json.Unmarshal(jsonStr, &payload)
+		for k, v := range eaSearch {
+			str, ok := v.(string)
+			if !ok {
+				log.Printf("Cannot marshal EA Search attribute for '%s'\n", k)
+			} else {
+				payload["*"+k] = str
+			}
+		}
+		jsonStr, err = json.Marshal(payload)
+		if err != nil {
+			log.Printf("Cannot marshal EA's in the payload: '%s'", payload)
+			return nil
+		}
+	}
+
```

Comment:
Yes this is WIP as I need to fix unit tests, but I opened the PR for early review.

Thanks Ko! Looks great idea to me. Let me try this out.